VERSION

```diff
@@ -1 +1 @@
-PKG_VERSION="2.41.0"
+PKG_VERSION="2.41.0-bms"
```

Comment:
This breaks the RPM builds with the following:
```
+ rpmbuild --define '_source_filedigest_algorithm md5' --define 'package_version 2.41.0-bms' --define '_sourcedir /tmp/cache_run/golang' --define '_rpmdir /tmp/cache_run/golang' -ba pkg/rpm/google-cloud-ops-agent.spec
error: line 16: Illegal char '-' in: Version: 2.41.0-bms
```

According to the [Fedora Packaging Guildelines](https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_upstream_uses_invalid_characters_in_the_version), only "ASCII letters (upper and lower case), digits and periods" are allowed in RPM version strings.

You might want to apply the following edit to `pkg/rpm/build.sh`:
```diff
-  --define "package_version $PKG_VERSION" \
+  --define "package_version ${PKG_VERSION//-/.}" \
```
or even:
```diff
-  --define "package_version $PKG_VERSION" \
+  --define "package_version ${PKG_VERSION//[^a-zA-Z0-9.]/.}" \
```
We should probably make the latter change in `master`…

Reply:
This is now done. Please pull in the changes from `master`.

Reply:
Actually, I just rebased `bms-support` (the base branch) on top of `master`. There were a couple of conflicts that I resolved. This should be good to go.